### DISCUSSION TITLE
Font is too small for me to read.  How do I adjust it?

### DISCUSSION THREAD

**Discussion by jtreagan** [keepassxreboot/keepassxc#11414] [Q&A]
I just did a new install of Ubuntu 24.04 and along with it a new install of KeepassXC, version 2.7.8.  Now the font is so small I can't read it.  How do I adjust it?

**Comment by droidmonkey** [comment]
Recommend switching to the flatpak distribution. You are probably using snap right now

**Reply by jtreagan** [reply]
Will try that and let you know how it comes out.  Thanks.
